### PR TITLE
Add example of using an SSH tunnel to access the console in development deployments

### PR DESCRIPTION
**Pull request by jordanhendricks**
A small docs change to the how to run doc, for something I always have to remind myself how to do.

Rendered version of how-to-run.adoc with this PR [here](https://github.com/jordanhendricks/omicron/blob/30cb8363da71abcd32550465a6c1b0f07f5d19a8/docs/how-to-run.adoc).

### CODE REVIEW

File: docs/how-to-run.adoc [hunk 1]
```diff
@@ -433,7 +436,27 @@ Where did 192.168.1.20 come from?  That's the external address of the external
...
+If you set up a fake external network (method 2 in <<external-networking>>), one
+way to be able to access the console of your deployment is by setting up an SSH
+tunnel. Console access is required to use the CLI for device authentication.
+The following is an example of how to access the console with an SSH tunnel.
+
+Nexus serves the console, so first get a nexus IP from the instructions above.
+
+On the machine from which you which to access the console, set up an SSH tunnel.
+This is an example for the lab machine `dunkin`. Replace the hostname there as
+appropriate, as well as the nexus IP.
```
Comment (davepacheco) [inline docs/how-to-run.adoc:452]:
```suggestion
In our example, Omicron is running on the lab machine `dunkin`.  You'll usually want to set up the tunnel from the machine where you run your browser to the machine running Omicron.  In our case, we'd run this on the machine running the browser:
```

(I found some of the existing paragraph a little confusing)

File: docs/how-to-run.adoc [hunk 2]
```diff
@@ -433,7 +436,27 @@ Where did 192.168.1.20 come from?  That's the external address of the external
...
+
+```
+$ ssh -L 1234:192.168.1.22:80 dunkin.eng.oxide.computer
+```
+
```
Comment (davepacheco) [inline docs/how-to-run.adoc:457]: I'd add:

> This configures `ssh` to bind to TCP port 1234 on the machine running the browser, forward packets through the ssh connection, and redirect them to 192.168.1.22 port 80 _as seen from the other side of the connection._

**Review by davepacheco — APPROVED** [review]

File: docs/how-to-run.adoc (hunk 2 above)
Comment (jordanhendricks) [reply docs/how-to-run.adoc:458]: done in f3ca0b4

File: docs/how-to-run.adoc (hunk 1 above)
Comment (jordanhendricks) [reply docs/how-to-run.adoc]: added something similar in f3ca0b4

**Pull request merged**